### PR TITLE
Ben/detox fix

### PR DESCRIPTION
**Pull request by bionara**
# Description

Reverts some suspish code - local build succeeds. I suspect the erroneous Buttons/index.ts (uppercase B) was responsible for some previous detox circle ci issues which is promising after develop is fixed.

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [ ] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

_Please attach screenshots showing the change if relevant_

## Checklist (for submission)

- [ ] Analytics/tracking events have been added (if needed)

## Checklist (for reviewers)

- [ ] Checked against Figma designs (if relevant)
- [ ] Checked that data has been successfully saved to the backend (if relevant)

## Out of scope and potential follow-up

_Are there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?_

### CODE REVIEW

File: ios/Covid.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -305,7 +304,6 @@
 			buildActionMask = 2147483647;
 			files = (
 				F90451F5267A4B6D0055A8AB /* main.jsbundle in Resources */,
```
Comment (aikewoody) [inline ios/Covid.xcodeproj/project.pbxproj:306]: Wasn't it this file that was added? And should be removed to revert the change

Comment (bionara) [reply ios/Covid.xcodeproj/project.pbxproj:306]: specifically these two lines - they are xcode config for "hooks" to build the js bundle. Will no doubt be reintroduced for detox circle CI.

**Review by selvach — DISMISSED** [review]
Removed the anniversary modal that crept back in....

File: ios/Covid.xcodeproj/project.pbxproj (hunk 1 above)
Comment (aikewoody) [reply ios/Covid.xcodeproj/project.pbxproj:306]: But why remove `SofiaPro-UltraLight.otf` file here?

Comment (bionara) [reply ios/Covid.xcodeproj/project.pbxproj]: ammended

**Review by selvach — APPROVED** [review]

**Review by aikewoody — APPROVED** [review]
